pandera/schemas.py

```diff
@@ -465,8 +478,11 @@ def validate(
 
             for column in columns:
                 is_schema_col = column in expanded_column_names
-                if self.strict and not is_schema_col:
-                    msg = f"column '{column}' not in DataFrameSchema {self.columns}"
+                if (self.strict is True) and not is_schema_col:
```

Comment:
Maybe I'm missing something but in this case I actually want to see if the value of strict is `True`, since having simply `if self.strict` would not get me to the second condition for filter.
```
>>>strict = 'filter'
>>>assert strict
>>>assert strict is True # will fail
```

Reply:
ah, I see! disregard my comment